src/System.Private.ServiceModel/dir.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.1.3.0</AssemblyVersion>
+    <AssemblyVersion>4.1.3.2</AssemblyVersion>
```

Comment:
To follow current versioning style, this should be `4.1.4.0` because UWP may have another servicing release in the future and that will need version `4.1.3.2`. So we need to reserve `4.1.3.x` for UWP.

Reply:
Makes sense.